### PR TITLE
[Research]UI: cluster drawer updates to meet research requirement.

### PR DESCRIPTION
**Pull request by hsinlun99**
# Description
1. Cluster Drawer UI change to meet the research requirement
2. Fix several typos in message keys

# Changes
1. cluster drawer
image
2. poi drawer (still needs to be updated)
image
image

# Notes
- highlight the corresponding poi marker is still unfinished.
- the "編輯" button is still unfinished.
image

# Checklist

- [x] I have performed a self-review of my own code
- [X] My changes generate no new warnings
- [X] I have checked my code and corrected any misspellings
- [X] Any changes to strings have been published to our translation tool
- [X] I conducted basic QA to assure all features are working
- [X] I requested code review from other team members

# Resolved Issues
part of https://github.com/CAMPUS-NYCU/smart-campus/issues/14

### CODE REVIEW

**Review by y-c-wang — CHANGES_REQUESTED** [review]
## PR Scope

Please make the next PR smaller and focus on **one** feature, e.g.
- change cluster drawer to contain the POIs within it
  - in this PR no additional changes need to be done, it is ok to use just a block with a string to denote the POI
- add chip of floor for POIs in cluster drawer
- add image placeholder for POIs in cluster drawer
- ...


## List changes in this PR

Please list all of the changes in this PR

**Comment by hsinlun99** [comment]
## Changes
### changes for new feature
- set the drawer height as 50vh for poi and cluster to meet research requirement
- update information displayed in cluster (e.g. delete cluster position, add [building name]現有回報資訊)
- add a list to contain pois in cluster
  - list `poi.data.target.category`, `poi.data.target.serial`, `poi.data.target.floor`, `poi.data.status.type`, `poi.data.status.value` to meet research requirement
  - list `poi.data.updateAt` or `poi.data.createdAt` (depends on whether the poi is updated) to meet the research requirement
  - add chip for floor, status type, and status value
  - add "編輯" button for research requirement (function is still unfinished)
  - add image place holder for research requirement (function is still unfinished)

### fix prior typos
- nonfunctional key error: correct bad to broken
- notCrowded key error: correct noSpaces to notCrowded
- badCleanliness key error: correct goodCleanliness to badCleanliness

File: src/components/Drawer/ClusterDrawer/index.tsx [hunk 1]
```diff
@@ -16,6 +17,100 @@ import {
...
+      onAction={(key) => alert(`TODO: highlight poi id: ${key}`)}
+    >
+      <ListboxItem
+        key={poi.id}
+        textValue={`list item of ${poi.id}`}
+        classNames={{
+          base: "border-1 border-secondary/50 h-[calc((50vh-100px)/4)] py-0",
+        }}
+      >
+        <div className="container flex flex-row justify-around space-x-0.5 py-0 h-[calc((50vh-100px)/4)]">
```
Comment (linyc0817) [inline src/components/Drawer/ClusterDrawer/index.tsx:52]: maybe using "h-fit" will be better. Let the height increase with the content.

File: src/components/Drawer/ClusterDrawer/index.tsx [hunk 2]
```diff
@@ -16,6 +17,100 @@ import {
...
+          {/* 主要資訊列 */}
+          <div className="flex flex-col shrink-0 justify-around basis-7/12">
+            <div className="flex flex-wrap flex-row space-x-1">
```
Comment (linyc0817) [inline src/components/Drawer/ClusterDrawer/index.tsx:55]: add text-align left to allow text move to the left when wrapping

File: src/locale/en-us/drawer.json [hunk 3]
```diff
@@ -70,15 +70,17 @@
   "clusterDrawer": {
     "buttons": {
       "add": "Add",
-      "cancel": "Cancel"
+      "cancel": "Cancel",
+      "edit": "Edit"
     },
     "content": {
       "texts": {
         "description": "Description: {{description}}",
```
Comment (linyc0817) [inline src/locale/en-us/drawer.json:78]: Did we still have description in our clusterDrawer content ?

**Review by linyc0817 — CHANGES_REQUESTED** [review]
Some style suggestion

File: src/components/Drawer/ClusterDrawer/index.tsx (hunk 2 above)
Comment (hsinlun99) [reply src/components/Drawer/ClusterDrawer/index.tsx]: If you mean the space before "無障礙停車區1" in this picture, it cannot be solved by adding text-align.
I fixed it in another way. You may check it again to see if it's satisfying :) 
image

Comment (linyc0817) [reply src/components/Drawer/ClusterDrawer/index.tsx]: 我的意思是 "空間/無障礙停車區" 或是 "無障礙停車區1" 這兩欄，如果內容太長導致換行，目前寫法會是置中，但應該要向左看齊。

**Review by linyc0817 — APPROVED** [review]
LGTM

**Pull request merged**